_gulp/config.js

```diff
@@ -29,14 +29,23 @@ module.exports = {
         }
     },
 
+    babel: {
+
+        src: 'js/theme.concat.es6.js',
```

Comment:
I would just override the `theme.concat.js` file after transpiling. No need for several files...?